<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AA2QtEdit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,1">
          <item>
           <widget class="QLabel" name="labelSort">
            <property name="text">
             <string>Sort by</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sortBy"/>
          </item>
          <item>
           <widget class="QComboBox" name="sortOrder"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="textFilter">
          <property name="placeholderText">
           <string>Filter names</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ClassEdit::CardListView" name="cardListView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="ClassEdit::CardView" name="cardView" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Save_File"/>
   <addaction name="actionLoad_Directory"/>
   <addaction name="actionSave_All_Changes"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Save_File">
   <property name="icon">
    <iconset resource="../../icons/icons.qrc">
     <normaloff>:/icon/game.svg</normaloff>:/icon/game.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Save File</string>
   </property>
  </action>
  <action name="actionLoad_Directory">
   <property name="icon">
    <iconset resource="../../icons/icons.qrc">
     <normaloff>:/icon/folder-people.svg</normaloff>:/icon/folder-people.svg</iconset>
   </property>
   <property name="text">
    <string>Load &amp;Folder</string>
   </property>
   <property name="toolTip">
    <string>Load Folder</string>
   </property>
  </action>
  <action name="actionSave_All_Changes">
   <property name="icon">
    <iconset resource="../../icons/icons.qrc">
     <normaloff>:/icon/save-all.svg</normaloff>:/icon/save-all.svg</iconset>
   </property>
   <property name="text">
    <string>Save All Changes</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../icons/icons.qrc">
     <normaloff>:/icon/window-close.svg</normaloff>:/icon/window-close.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClassEdit::CardListView</class>
   <extends>QListView</extends>
   <header>src/ui/cardlistview.h</header>
  </customwidget>
  <customwidget>
   <class>ClassEdit::CardView</class>
   <extends>QWidget</extends>
   <header>src/ui/cardview.h</header>
   <container>1</container>
   <slots>
    <slot>modelItemActivated(QModelIndex)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionLoad_Save_File</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadSaveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad_Directory</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cardListView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>cardView</receiver>
   <slot>modelItemActivated(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>781</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadSaveFile()</slot>
  <slot>loadDirectory()</slot>
  <slot>itemActivated(QModelIndex)</slot>
  <slot>quit()</slot>
 </slots>
</ui>
